src/contactForm.js

```diff
@@ -118,6 +118,9 @@ class RecrasContactForm {
             }
             contactForm[field.dataset.identifier].push(field.value);
         });
+        if (contactForm['boeking.datum']) {
+            contactForm['boeking.datum'] = RecrasDateHelper.datePartOnly(this.selectedDate);
```

Comment:
We should probably add some tests for this invalid values, valid values, empty field etc